test/test-tarantool/panic_on_broken_lsn.test.lua

```diff
@@ -0,0 +1,9 @@
+-- regression test for https://github.com/tarantool/tarantool-qa/issues/96
```

Comment:
1. I would link a particular problem. The QA issue is too general: it links two problems now and we maybe will add more ones on this kind to it (if we'll see them in CI). There is no relevant issue in test-run, because I just filed PR with the problem description: https://github.com/tarantool/test-run/pull/275 (not a best practice, sure).
2. I would rename the test case to something relevant to the problem (there is nothing about a broken snapshot here).
3. And the test case reproduces the problem with a small probability (without patching `multiprocessing/connection.py`), so I don't know, whether it ever worth to add it. At least we should make this fact clear.